frank-doc-doclet/src/main/java/org/frankframework/frankdoc/feature/Deprecated.java

```diff
@@ -16,14 +16,68 @@
 
 package org.frankframework.frankdoc.feature;
 
+import org.frankframework.frankdoc.model.DeprecationInfo;
+import org.frankframework.frankdoc.wrapper.FrankAnnotation;
+import org.frankframework.frankdoc.wrapper.FrankClass;
+import org.frankframework.frankdoc.wrapper.FrankDocException;
+import org.frankframework.frankdoc.wrapper.FrankMethod;
+
 public class Deprecated extends AbstractNonValuedFeature {
 	private static final Deprecated INSTANCE = new Deprecated();
+	private static final String DEPRECATED_ANNOTATION_CLASSNAME = "java.lang.Deprecated";
+	private static final String CONFIG_WARNING_ANNOTATION_CLASSNAME = "org.frankframework.configuration.ConfigurationWarning";
 
 	public static Deprecated getInstance() {
 		return INSTANCE;
 	}
 
 	private Deprecated() {
-		super("java.lang.Deprecated", "@deprecated");
+		super(DEPRECATED_ANNOTATION_CLASSNAME, "@deprecated");
 	}
+
+	public DeprecationInfo getInfo(FrankClass clazz) throws FrankDocException {
+		FrankAnnotation deprecationAnnotation = clazz.getAnnotation(DEPRECATED_ANNOTATION_CLASSNAME);
+		FrankAnnotation configWarningAnnotation = clazz.getAnnotation(CONFIG_WARNING_ANNOTATION_CLASSNAME);
+
+		return getInfo(deprecationAnnotation, configWarningAnnotation);
+	}
+
+	public DeprecationInfo getInfo(FrankMethod method) throws FrankDocException {
+		FrankAnnotation deprecationAnnotation = method.getAnnotation(DEPRECATED_ANNOTATION_CLASSNAME);
+		FrankAnnotation configWarningAnnotation = method.getAnnotation(CONFIG_WARNING_ANNOTATION_CLASSNAME);
+
+		return getInfo(deprecationAnnotation, configWarningAnnotation);
+	}
+
+	private DeprecationInfo getInfo(FrankAnnotation deprecationAnnotation, FrankAnnotation configWarningAnnotation) throws FrankDocException {
```

Comment:
Waar gooi je hier een FrankDocException?

Reply:
Dat stond in de signature van de `FrankAnnotation`, terwijl in de onderligende implementatie nergens een exceptie gooide. Verwijderd. Dit patroon komt wel vaker voor in de codebase